src/Controller/AccountController.java

```diff
@@ -129,8 +129,10 @@ public void validateNumber()
     {
         if (account_no.getText().trim().isEmpty())
             this.submit.setDisable(true);
-        else
-            this.handleChange();
+        else {
+        	this.submit.setStyle("-fx-text-box-border:red;");
```

Comment:
The indentation of these lines is incorrect.  They also have both spaces and tabs.  You need to fix the indentation and use only tabs.